bin/bgv2json

```diff
@@ -3,6 +3,23 @@
 require 'seafoam'
 require 'json'
 
+def prepare_json(object)
+  case object
+  when Float
+    if object.nan?
+      'NaN'
```

Comment:
Who’s going to be consuming this JSON? Is providing a non-numeric value better than e.g. omitting the key?

Reply:
I guess you're really saying I should add a test! I refactored in #17 to make that easier to do, and will rebase on top of that when it's merged.